addons/mail/models/mail_render_mixin.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+class MailRenderMixin(models.AbstractModel):
+
+    _name = 'mail.render.mixin'
+    _description = 'Mail Render Mixin'
+
+    model_object_field = fields.Many2one(
```

Comment:
StylingNote: in Odoo we generally try to lessen line count by stacking a bit parameters on lines, while trying to stay below ~100 / 120 characters. I would for example vote for
```
model_object_field = fields.Many2one(
     'ir.model.fields', string="Field", store=False,
     help="Very long help message")
```